src/server/services/email.ts

```diff
@@ -42,7 +42,11 @@ export class MailerNode implements Mailer {
     const emailHTML = `Your OTP is <b>${otp}</b>. It will expire in ${Math.floor(
       otpExpiry / 60,
     )} minutes.
-    Please use this to login to your go.gov.sg account. <p>If your OTP does not work, please request for a new OTP.</p>`
+    Please use this to login to your GoGovSG account.
+    <p>If your OTP does not work, please request for a new OTP at https://go.gov.sg on the internet.</p>
+    <p>If you did not make this request, you may ignore this email.</p>
+    <p>This login attempt was made from the IP: ${ip}. If you did not attempt to log in to GoGovSG, you may choose to investigate this IP address further.</p>
+    <p>The GoGovSG Support Team</p>`
```

Comment:
Could we perhaps avoid references to GoGovSG here? It would be helpful for those who may wish to reuse our codebase

Reply:
@yong-jie also elsewhere in that text, not just line 49 -_-

Reply:
what about the go.gov.sg url? that one also has to go i assume?